cmd/create/cluster/cmd.go

```diff
@@ -428,9 +444,7 @@ func run(cmd *cobra.Command, _ []string) {
 			options[i] = setSubnetOption(subnetID, availabilityZone)
 			if subnetsProvided {
 				for _, subnetArg := range subnetIDs {
-					if subnetArg == subnetID {
-						optionDefaults = append(optionDefaults, setSubnetOption(subnetID, availabilityZone))
-					}
+					defaultOptions = append(defaultOptions, setSubnetOption(subnetArg, availabilityZone))
```

Comment:
Doesn't this make it so that _all_ subnets are selected, regardless of which were sent in the CLI?

Reply:
@vkareh why? only those passed via the CLI (i.e. `subnetArg`) are selected..

Reply:
duh, makes sense :stuck_out_tongue_closed_eyes: